typescript/src/start-script.ts

```diff
@@ -65,7 +74,7 @@ if (Array.isArray(argv._)) {
   protoFiles.push(argv._);
 }
 
-const commonProtoPath = argv.commonProtoPath || GOOGLE_GAX_PROTOS_DIR;
+const commonProtoPath = argv.commonProtoPath || googleGaxProtosDir;
```

Comment:
I curious why do we make change here. Looks like GOOGLE_GAX_PROTOS_DIR is constant variable.

Reply:
We used to use `CAPITAL_LETTERS` for constants in some places but then refactored them all to regular `camelCase`, but this one was forgotten :)